yarn.lock

```diff
@@ -211,10 +211,6 @@
   version "9.2.1-sha-b0424a351"
   resolved "https://codeload.github.com/angular/material2-docs-content/tar.gz/66f276a9526a373b4c252370514f10de130561f0"
 
-"@angular/components-examples@angular/material2-docs-content#master":
```

Comment:
This shouldn't be removed. It looks like you rebuilt the entire lock file? If so, then you'll need to run `npm run build:content`.

Reply:
I try run 'yarn run build:content' and 'yarn upgrade @angular/components-examples', but `yarn.lock` has not changed

Reply:
AH OK, that makes sense. The diff didn't show that the one above it was for the same package.